pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -70,7 +70,7 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 				phaseDisplay := fmt.Sprintf("Phase %s (%s strategy) [NOT ACTIVE]", phase.Name, phase.Strategy)
 				phaseBranchName := planBranchName.AddBranch(phaseDisplay)
 				for _, steps := range plan.Phases {
-					stepsDisplay := fmt.Sprintf("Step %s (%s strategy) [NOT ACTIVE]", steps.Name, steps.Strategy)
```

Comment:
🙈 I am so confused, it's named steps but it is of type phase

Reply:
Yeah, this section needed some attention as well - I've fixed it up so that it shows the right output for phases and steps.